hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -63,3 +63,8 @@ set_rule_status -rule {W_CNTL} -status {Waived}                              \
 set_rule_status -rule {W_MASYNC} -status {Waived}        \
   -expression {(Driver=~"*u_pinmux_aon.dio_pad_attr_*")} \
   -comment {PAD Attributes are static signals.}
+
+# JTAG en
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived} \
+  -expression {(Signal=~"*.u_pinmux_strap_sampling.jtag_en*")} \
+  -comment {jtag_en changes clock mux. Can be assumed as static signal}
```

Comment:
actually this is something that the tester environment can dynamically change during certain life cycle states.
Could you just update the comment to say that it is tester responsibility to ensure no jtag transactions are ongoing before they make a change? This is fully under tester control.